psl-core/src/main/java/org/linqs/psl/evaluation/statistics/Evaluator.java

```diff
@@ -104,6 +104,24 @@ public double getNormalizedRepMetric() {
         return value;
     }
 
+    /**
+     * Combine getMaxRepMetric() with isHigherRepBetter() so that the values from getNormalizedRepMetric are always
+     * less than or equal to this value.
+     */
+    public double getNormalizedMaxRepMetric() {
+        double value = getMaxRepMetric();
+        if (!isHigherRepBetter()) {
+            value = -value;
+        }
+
+        return value;
+    }
+
+    /**
+     * The maximum (i.e. the best) value that the representative (rep) can take for this evaluator.
+     */
+    public abstract double getMaxRepMetric();
```

Comment:
The name is too confusing (people may not all see the comment).
Maybe: `getBestRepScore`?